### PR TITLE
2019 Survey announcements

### PR DESCRIPTION
**Pull request by GregSutcliffe**
One for the blog (so it appears in Foreman dashboards :P) and one for the website.

### CODE REVIEW

**Comment by GregSutcliffe** [comment]
Checks passed and I need to get this released, so I'm going to abuse my privileges and merge this if no one beats me to it in the next half-hour :)

File: _includes/alerts.html [hunk 1]
```diff
@@ -1,8 +1,7 @@
 <div class='tease'>
   <p>
   <a>Conference season is here! Join us at FOSDEM &amp; Config Management Camp, 2nd - 6th February 2019</a>
-  <a>We'd also love to hear your Foreman story at Configuration Management Camp!</a>
-    <br/>
-    <a href='https://cfp.cfgmgmtcamp.be'>Submit your talk proposal here! (closes 1st Dec)</a>
+  <br/>
+  <a href='https://da.gd/tfm_survey'>The 2019 Foreman Community Survey is now open! Click here to participate!</a>
```
Comment (ekohl) [inline _includes/alerts.html]: I prefer full URLs if they're links anyway. URL shorteners are for humans :)

Comment (GregSutcliffe) [reply _includes/alerts.html]: It's about 1 billion characters of a Google Forms URL and I hate it :P but I'll put it in if you insist...

**Review by ekohl — APPROVED** [review]

**Comment by GregSutcliffe** [comment]
Ty for the review @ekohl

**Pull request merged**